packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -0,0 +1,386 @@
+
+      await processUtils.run(
+        lipoCommand,
+        workingDirectory: outputDirectory.path,
+        allowReentrantFlutter: false,
+      );
+
+      // Create simulator framework.
+      final Directory simulatorFlutterFrameworkDirectory = iPhoneSimulatorDirectory.childDirectory(flutterFrameworkFileName);
+      final File simulatorFlutterFrameworkBinary = simulatorFlutterFrameworkDirectory.childFile('Flutter');
+      copyDirectorySync(fatFlutterFrameworkCopy, simulatorFlutterFrameworkDirectory);
+
+      lipoCommand = <String>['xcrun', 'lipo', fatFlutterFrameworkBinary.path, '-thin', 'x86_64', '-output', simulatorFlutterFrameworkBinary.path];
+
+      await processUtils.run(
+        lipoCommand,
+        workingDirectory: outputDirectory.path,
+        allowReentrantFlutter: false,
+      );
+
+      // Create XCFramework from iOS and simulator frameworks.
+      final List<String> xcframeworkCommand = <String>[
+        'xcrun',
+        'xcodebuild',
+        '-create-xcframework',
+        '-framework', armFlutterFrameworkDirectory.path,
+        '-framework', simulatorFlutterFrameworkDirectory.path,
+        '-output', modeDirectory
+          .childFile('Flutter.xcframework')
+          .path
+      ];
+
+      await processUtils.run(
+        xcframeworkCommand,
+        workingDirectory: outputDirectory.path,
+        allowReentrantFlutter: false,
+      );
+    }
+
+    if (!argResults['universal']) {
+      fatFlutterFrameworkCopy.deleteSync(recursive: true);
+    }
+    status.stop();
+  }
+
+  Future<void> _produceAppFramework(BuildMode mode, Directory iPhoneModeDirectory, Directory modeDirectory) async {
+    const String appFrameworkName = 'App.framework';
+    final Directory destinationAppFrameworkDirectory = modeDirectory.childDirectory(appFrameworkName);
+
+    if (mode == BuildMode.debug) {
+      final Status status = logger.startProgress(' ├─Add placeholder App.framework for debug...', timeout: timeoutConfiguration.fastOperation);
+      await createStubAppFramework(destinationAppFrameworkDirectory);
+      status.stop();
+    } else {
+      await _produceAotAppFrameworkIfNeeded(mode, iPhoneModeDirectory, destinationAppFrameworkDirectory);
+    }
+
+    final FlutterProject flutterProject = FlutterProject.current();
+    final File sourceInfoPlist = flutterProject.ios.hostAppRoot.childDirectory('Flutter').childFile('AppFrameworkInfo.plist');
+    final File destinationInfoPlist = destinationAppFrameworkDirectory.childFile('Info.plist')..createSync(recursive: true);
+
+    destinationInfoPlist.writeAsBytesSync(sourceInfoPlist.readAsBytesSync());
+
+    final Status status = logger.startProgress(' ├─Assembling Flutter resources for App.framework...', timeout: timeoutConfiguration.slowOperation);
+    await bundleBuilder.build(
+      platform: TargetPlatform.ios,
+      buildMode: mode,
+      // Relative paths show noise in the compiler https://github.com/dart-lang/sdk/issues/37978.
```

Comment:
Its not the relative paths, but the bundleBuilder needs an absolute path to correctly lookup the package scheme

Reply:
Is there anything in any of the dart docs I can add for the upstream functions?

Reply:
I think linking to the above issue is enough